src/mtev_reverse_socket.c

```diff
@@ -2237,6 +2256,29 @@ mtev_reverse_socket_connection_shutdown(const char *address, int port) {
   pthread_mutex_unlock(&reverses_lock);
   return success;
 }
+mtev_boolean
+mtev_connection_do(const char *address, int port, void (*cb)(mtev_connection_ctx_t *, reverse_socket_t *, void *), void *closure) {
+  mtev_hash_iter iter = MTEV_HASH_ITER_ZERO;
+  mtev_boolean invoked = mtev_false;
+  char remote_str[INET6_ADDRSTRLEN + 1 + 5 + 1];
```

Comment:
What is `INET6_ADDRSTRLEN + 1 + 5 + 1` from ?

Reply:
longest name + : + longest port + \0